pkg/ccloudv2/kafkarest.go

```diff
@@ -341,3 +366,29 @@ func (c *KafkaRestClient) UpdateKafkaTopicPartitionCount(topicName string, updat
 func (c *KafkaRestClient) GetKafkaTopic(topicName string) (kafkarestv3.TopicData, *http.Response, error) {
 	return c.TopicV3Api.GetKafkaTopic(c.kafkaRestApiContext(), c.ClusterId, topicName).Execute()
 }
+
+// GetShareGroupTopicNames extracts unique topic names from assigned topic partitions
+func (c *KafkaRestClient) GetShareGroupTopicNames(shareGroup kafkarestv3.ShareGroupData) string {
+	topicPartitions := shareGroup.GetAssignedTopicPartitions()
+	if len(topicPartitions) == 0 {
+		return "None"
+	}
+
+	// Use a map to collect unique topic names
+	topicSet := make(map[string]bool)
+	for _, tp := range topicPartitions {
+		topicSet[tp.GetTopicName()] = true
+	}
+
+	if len(topicSet) == 0 {
+		return "None"
+	}
```

Comment:
```suggestion
```

Same reason as above: better to return `""`, in which case we don't need this block.